apis/kubekey/v1alpha2/cluster_types.go

```diff
@@ -175,6 +175,7 @@ type RegistryConfig struct {
 	RegistryMirrors    []string             `yaml:"registryMirrors" json:"registryMirrors,omitempty"`
 	InsecureRegistries []string             `yaml:"insecureRegistries" json:"insecureRegistries,omitempty"`
 	PrivateRegistry    string               `yaml:"privateRegistry" json:"privateRegistry,omitempty"`
+	PrivateRegistryIp  string               `yaml:"privateRegistryIp" json:"privateRegistryIp,omitempty"`
```

Comment:
Before,  the value of `privateRegistry` can also be set to an IP address. Would this field be ambiguous？

Reply:
The `privateRegistry ` is an optional field, and the registry recommends using the domain to configure，use IP will have many permission problems

Maybe the  field  name needs to be adjusted